configuration/sitemaps.md

```diff
@@ -270,11 +274,13 @@ Selection item=LR_TV_Channel label="TV Channel" mappings=[0="DasErste", 1="BBC O
 Setpoint item=<itemname> [label="<labelname>"] [icon="<iconname>"] minValue=<min value> maxValue=<max value> step=<step value>
 ```
 
-A special Switch-like element to increase or decrease the value of an item.
+A special Switch-like element that may be used to increase or decrease the value of an item.
 The element is often used to gradually change a number item.
 
+<!-- TODO: The sentence above is not clear and needs to be rewritten. -->
```

Comment:
I'm not even sure if a setpoint or slider can be used with anything other than a Number Item. Would you please do a quick test?

If nothing surprising comes up you can combine both sentences and state "of a Number Item" for both elements.

Reply:
I have a development system here, so no problem to do a test.  But what do you suggest?  Is your point to test something like a Switch Item to see if the Setpoint can be used to control it?